src/project/Manager.java

```diff
@@ -42,31 +42,21 @@ public void setHotel(Hotel hotel) {
         this.hotel = hotel;
     }
 
-    /**
-     * Create new booking
-     *
-     * @param newBooking booking interval
-     * @param countBeds  the requested number of beds
-     * @param days       number of booked days
-     * @return the result of booking - successful or not
-     */
+    public Room tryReservation(Booking newBooking, Bed beds) {
```

Comment:
It's better to give commands to the classes f.e. `makeReservation()` and then expect to either succeed of fail with an exception. In this case returning null represents a failed operation.

Reply:
If the method is not used, remove it